src/sentry/eventstream/kafka/backend.py

```diff
@@ -119,7 +53,7 @@ def _send(self, project_id, _type, extra_data=(), asynchronous=True):
                 topic=self.publish_topic,
                 key=key.encode('utf-8'),
                 value=json.dumps(
-                    (EVENT_PROTOCOL_VERSION, _type) + extra_data
+                    (self.EVENT_PROTOCOL_VERSION, _type) + extra_data
```

Comment:
This feels a bit like this abstraction leaked out of the `SnubaProtocolEventStream`?